app/scripts/controllers/projects.js

```diff
@@ -16,14 +38,52 @@ angular.module('openshiftConsole')
     $scope.showGetStarted = false;
     $scope.canCreate = undefined;
 
+    var sortProjects = function() {
+      // TODO: currentField is renamed in angular-patternfly 3.0
+      var sortID = _.get($scope, 'sortConfig.currentField.id', 'metadata.name');
+      var adjustedSortOrder = $scope.sortConfig.isAscending;
+
+      // Sort order must be reversed if using timestamp because the info shown to user is the relative timestamp, not the absolute
+      if (sortID === 'metadata.creationTimestamp') {
+        adjustedSortOrder = !adjustedSortOrder;
+      }
+
```

Comment:
May just be my opinion, but I think you could inline `var sortOrder = adjustedSortOrder ? 'asc' : 'desc';` into the `_.sortByOrder` fn's third arg and eliminate the additional `sortOrder` variable.  It may be easier to follow.  `}, [adjustedSortOrder ? 'asc' : 'desc']);`